### PR TITLE
[improvement](be) add a name for be jvm

### PR DESCRIPTION
**Pull request by zhangstar333**
# Proposed changes

before jps will not display the name of BE,  so don't know it's the Doris be
then maybe killed by user.

Issue Number: close #xxx
Snipaste_2023-03-09_11-38-15

## Problem summary

Describe your changes.

## Checklist(Required)

* [ ] Does it affect the original behavior
* [ ] Has unit tests been added
* [ ] Has document been added or modified
* [ ] Does it need to update dependencies
* [ ] Is this PR support rollback (If NO, please explain WHY)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Review by Gabriel39 — DISMISSED** [review]
LGTM

File: be/src/util/jni-util.cpp [hunk 1]
```diff
@@ -73,10 +73,13 @@ void FindOrCreateJavaVM() {
         auto classpath = GetDorisJNIClasspath();
         std::string heap_size = fmt::format("-Xmx{}", config::jvm_max_heap_size);
         std::string log_path = fmt::format("-DlogPath={}/log/udf-jdbc.log", getenv("DORIS_HOME"));
+        std::string jvm_name = fmt::format("-Dsun.java.command={}", "DORIS_BE");
```
Comment (adonis0147) [inline be/src/util/jni-util.cpp:76]:
```suggestion
        std::string jvm_name = fmt::format("-Dsun.java.command={}", "DorisBE");
```

**Comment by zhangstar333** [comment]
run buildall

**Review by adonis0147 — APPROVED** [review]
LGTM

**Comment by hello-stephen** [comment]
TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 34.46 seconds
 stream load tsv:          473 seconds loaded 74807831229 Bytes, about 150 MB/s
 stream load json:         38 seconds loaded 2358488459 Bytes, about 59 MB/s
 stream load orc:          76 seconds loaded 1101869774 Bytes, about 13 MB/s
 stream load parquet:          31 seconds loaded 861443392 Bytes, about 26 MB/s
 https://doris-community-test-1308700295.cos.ap-hongkong.myqcloud.com/tmp/20230309104340_clickbench_pr_111414.html

**Pull request merged**